scala-interpreter/src/main/scala/org/apache/toree/kernel/interpreter/scala/ScalaInterpreter.scala

```diff
@@ -48,7 +48,7 @@ class ScalaInterpreter(private val config:Config = ConfigFactory.load) extends I
 
    protected val lastResultOut = new ByteArrayOutputStream()
 
-   protected val multiOutputStream = MultiOutputStream(List(Console.out, lastResultOut))
+   protected val multiOutputStream = MultiOutputStream(List(lastResultOut))
```

Comment:
I think the multiOutputStream can be completely removed rather than using it to wrap just one output stream. Instead, lastResultOut should be passed to IMain instead of multiOutputStream.